libs/content-search-toolkit/src/queries/search.ts

```diff
@@ -11,17 +11,31 @@ export const searchQuery = ({
   tags = [],
   countTag = '',
   countTypes = false,
+  fuzzy = false,
+  fuzzyFactor = 1,
 }: SearchInput) => {
   const should = []
   const must: TagQuery[] = []
   let minimumShouldMatch = 1
+  const wordLengthRequiredForFuzz = 5
+
+  if (fuzzy) {
+    queryString = queryString
+      .split(' ')
+      .map((term) =>
+        term.length >= wordLengthRequiredForFuzz
+          ? `${term}~${fuzzyFactor}`
+          : term,
+      )
+      .join(' ')
+  }
 
   should.push({
     simple_query_string: {
       query: queryString,
       fields: ['title.stemmed^15', 'title.compound', 'content.stemmed^5'],
       analyze_wildcard: true,
-      default_operator: 'and',
+      default_operator: 'or',
```

Comment:
When `OR`-ing together multiple terms, how is the ranking of result hits? Does it rank hits higher if it matches all terms? I.e. we want the topmost search result to be the most relevant based on all the terms.

Reply:
So basically "no, but it's not that simple"

Reply:
Actually, this is hasty and it's been rightfully pointed out that there are problems with `OR` which I didn't catch on. There are other ways and methods to be explored.
Closing this for now